godotopenxrmeta/build.gradle

```diff
@@ -3,44 +3,38 @@ plugins {
 }
 
 android {
-    compileSdk 31
+    compileSdk versions.compileSdk
 
     defaultConfig {
-        minSdk 19
-        targetSdk 31
-        versionCode 1
-        versionName "1.0"
+        minSdk versions.minSdk
+        targetSdk versions.targetSdk
+        versionCode versions.versionCode
+        versionName versions.versionName
 
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         ndk {
             abiFilters "arm64-v8a"
         }
     }
 
+    namespace = "org.godotengine.openxrloaders.meta"
+
     buildTypes {
         debug {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
-        packagingOptions {
-            doNotStrip '**/*.so'
-        }
     }
 
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+    packagingOptions {
+        doNotStrip '**/*.so'
     }
-    buildFeatures {
-        viewBinding true
+
+    compileOptions {
+        sourceCompatibility versions.javaVersion
+        targetCompatibility versions.javaVersion
     }
 }
 
 dependencies {
-    implementation "androidx.legacy:legacy-support-v4:1.0.0"
-
-    compileOnly 'io.github.m4gr3d:godot:4.0.0.beta3'
-    testImplementation 'junit:junit:4.+'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    compileOnly "io.github.m4gr3d:godot:$versions.godotLibVersion"
```

Comment:
What about compiling this with a custom compiled godot lib?

Previously (before this [commit](https://github.com/GodotVR/godot_openxr_loaders/commit/7bbd37c89fc1dcc9ca9311421d88c8af85474685#diff-3c2e668dd5d5cee7472c1fb57a9b21a07ed797e3324576aef1d90976d263f41cL42)), this looked like:

```
compileOnly files('src/godot/godot-lib.release.aar'
```

Although, I suppose contributors can hack it and change this line :-)

Reply:
Yes for a custom version of the lib, you'd have to change the line back to what it was previously. 
The current change is to make it easier to keep up-to-date with lib update by leveraging maven.

Reply:
Is this really important though? We're not actually including the library, it will use whatever is exported by the custom build you're using. So unless there is some API breakage on the tiny bit of the godot-lib logic that is used here, it should just work?